File: ServerINes/Features/Dashboard/DTOs/DashboardStatsDto.cs
namespace INest.Features.Dashboard.DTOs
{
    public class DashboardStatsDto
    {
        public int TotalItemsCount { get; set; }
        public int TotalLocationsCount { get; set; }
        public int ExpiredRemindersCount { get; set; }
        public int ExpiringWarrantiesCount { get; set; }
        public int LentItemsCount { get; set; }
        public int ArchivedAndSoldItemsCount { get; set; }
    }
}

File: ClientINes/src/app/features/dashboard/dtos/dashboard-stats.dto.ts
import { AttentionItemDto } from "../../../features/item/dtos/item-attention.dto";

export interface DashboardStatsDto {
  totalItemsCount: number;
  totalLocationsCount: number;
  expiredRemindersCount: number;
  expiringWarrantiesCount: number;
  lentItemsCount: number;
  soldItemsCount: number;
  activeRemindersCount?: number;
  attentionItems?: AttentionItemDto[];
}